### DISCUSSION TITLE
Toolset for upgrading from Bootstrap 4.x to 5.2

### DISCUSSION THREAD

**Discussion by altdelnow** [twbs/bootstrap#38653] [General]
Hi All,
I am looking for a tool(s) which can automate bootstrap upgrade from4.x to 5.2.
Thanks in advance
Rajeshwar Singh

**Comment by coliff** [comment]
I've created a Bootstrap 5 Migrate Tool command-line script which scans all pages/templates in a folder and:

- Replaces all the Bootstrap 4 classes with Bootstrap 5 classes.
- Renames all the data attributes (such as data-toggle and data-target) to the new bs-prefixed names
- Replaces deprecated components (such as `.jumbotron` and `.media`) with the relevant utility classes
- Updates all Bootstrap v4 CDN links (jsdelivr.net, unpkg.com and stackpath.bootstrapcdn.com) to latest version of Bootstrap v5.

It's free and open-source: https://github.com/coliff/bootstrap-5-migrate-tool
